setup.cfg

```
@@ -35,8 +35,11 @@ python_requires = >=2.7,!=3.0,!=3.1,!=3.2,!=3.3
 setup_requires =
     setuptools>=40.8.0
     wheel
+
+# colorama upper bounds pin is temporary fix
+# due to https://github.com/sarugaku/vistir/issues/98
 install_requires =
-    colorama>=0.3.4
+    colorama>=0.3.4,<=0.4.1
```

FYI this is apparently resolved now:
```suggestion
    colorama>=0.3.4,!=0.4.2
```

---

Done